drivers/SmartThings/zigbee-button/src/zigbee-multi-button/linxura/init.lua

```diff
@@ -0,0 +1,129 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local IASZone = (require "st.zigbee.zcl.clusters").IASZone
+local constants = require "st.zigbee.constants"
+local clusters = require "st.zigbee.zcl.clusters"
+local Groups = clusters.Groups
+local log = require "log"
+local device_management = require "st.zigbee.device_management"
+local zcl_commands = require "st.zigbee.zcl.global_commands"
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+
+local BUTTON1_HELD = 0x0001
+local BUTTON1_PUSHED = 0x0003
+local BUTTON1_DOUBLE = 0x0005
+local BUTTON2_HELD = 0x0007
+local BUTTON2_PUSHED = 0x0009
+local BUTTON2_DOUBLE = 0x000B
+local BUTTON3_HELD = 0x000D
+local BUTTON3_PUSHED = 0x000F
+local BUTTON3_DOUBLE = 0x0011
+local BUTTON4_HELD = 0x00013
+local BUTTON4_PUSHED = 0x00015
+local BUTTON4_DOUBLE = 0x0017
+
+local BUTTON1 = "button1"
+local BUTTON2 = "button2"
+local BUTTON3 = "button3"
+local BUTTON4 = "button4"
+
+
+local LINXURA_BUTTON_FINGERPRINTS = {
+    { mfr = "Linxura", model = "Smart Controller" }
+}
+
+local configuration = {
+    {
+        cluster = IASZone.ID,
+        attribute = IASZone.attributes.ZoneStatus.ID,
+        minimum_interval = 0,
+        maximum_interval = 3600,
+        data_type = IASZone.attributes.ZoneStatus.base_type,
+        reportable_change = 1
+    }
+}
+local is_linxura_button = function(opts, driver, device)
+    for _, fingerprint in ipairs(LINXURA_BUTTON_FINGERPRINTS) do
+        if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+        return true
+        end
+    end
+    return false
+end
+
+local function present_value_attr_handler(driver, device, zone_status, zb_rx)   
+    log.info("present_value_attr_handler The current value is: ", zone_status.value)
+    local status = zone_status
+    
+
+    local additional_fields = {
+        state_change = true
+    }
+    local event
+    local mod = status.value % 6
+    if mod == 1 then
+        event = capabilities.button.button.pushed(additional_fields)
+    elseif mod == 3 then
+        event = capabilities.button.button.double(additional_fields)
+    elseif mod == 5 then
+        event = capabilities.button.button.held(additional_fields)
+    else
+        return false
```

Comment:
I think you just want a void `return` here. Not sure how the runner would interpret a `false` value being returned.